thredded.gemspec

```diff
@@ -22,9 +23,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'htmlentities'
   s.add_dependency 'kaminari'
   s.add_dependency 'nokogiri'
-  s.add_dependency 'q'
   s.add_dependency 'rails', '>= 4.0.0'
   s.add_dependency 'rb-gravatar'
+  s.add_dependency 'activejob_backport' if Thredded::RailsVersion.less_than_4_2?
```

Comment:
Ha! Yeah I knew something didn't feel right. Should have listened to my gut
instead of my 2-beer-deep brain

On Saturday, April 2, 2016, Gleb Mazovetskiy <notifications@github.com>
wrote:

> In thredded.gemspec
> <https://github.com/thredded/thredded/pull/177#discussion_r58291318>:
>
> >    s.add_dependency 'rails', '>= 4.0.0'
> >    s.add_dependency 'rb-gravatar'
> > +  s.add_dependency 'activejob_backport' if Thredded::RailsVersion.less_than_4_2?
>
> Yeah, it doesn't work like this, gem dependencies are determined at the
> time of building the gem
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/177/files/591bcbf7f1e0a139a61f256d7326eedafa3e72f6#r58291318>
>